refreshVersions/src/main/kotlin/de/fayard/versions/internal/UsedDependenciesWriting.kt

```diff
@@ -3,13 +3,22 @@ package de.fayard.versions.internal
 import de.fayard.versions.extensions.isRootProject
 import org.gradle.api.Project
 import org.gradle.api.artifacts.Dependency
+import org.gradle.api.initialization.Settings
 import org.gradle.api.internal.artifacts.dependencies.AbstractDependency
+import java.lang.Appendable
 
-internal fun Project.readDependenciesUsedInBuildSrc(): Sequence<Dependency> {
+internal fun Project.readExtraUsedDependencies(): Sequence<Dependency> {
```

Comment:
the naming is not really clear..

Reply:
Renamed to `readPluginsAndBuildSrcDependencies`.